pages/quickstart/setting-up-a-local-environment-with-dive-cli.mdx

```diff
@@ -0,0 +1,165 @@
+import { Callout } from "nextra/components";
+
+# Setup Local Environment Using DIVE Cli
+
+## Introduction
+
+Welcome to the DIVE Command-Line Interface (CLI), This guide will help you get up and running with DIVE CLI, the all-in-one tool designed to simplify blockchain development, set up local environments, and enable seamless cross-chain communication.
```

Comment:
Remove "Welcome to the DIVE Command-Line Interface (CLI), ".

This is a great opportunity to add some background information to the tool and HugoByte. ie:

"[DIVE](https://github.com/HugoByte/DIVE) is an [organic community-driven tool](https://cps.icon.community/proposals/bafybeiea4bs4wb3donxhhvu2gjygqrdjrwyrxzajz52aamwg4yni2a2ltm) designed to simplify blockchain development, set up local environments, and enable seamless cross-chain communication. DIVE is built by [HugoByte](https://tracker.icon.community/address/hx31c429608c74501de13df5fea30bf1674460be50), a validator on the ICON network.

> **_NOTE:_** DIVE is currently in beta. Things may break. We are always looking to improve the tool and add new features, so if you have any issues or ideas please post them to the [DIVE issue tracker](https://github.com/HugoByte/DIVE/issues).

This guide will help you get up and running with DIVE CLI. It will walk you through the steps of how to set up an ICON node, EVM Hardhat node, and a relay to pass messages between them.
"